src/marine/godae/CMakeLists.txt

```diff
@@ -9,8 +9,3 @@ COPY_FILES( "${programs}" ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_BINARY_DIR}/bin )
 install (PROGRAMS
   ${programs}
   DESTINATION bin)
-
```

Comment:
${programs} list is not getting built properly. The file names are missing the "godae_" prefix.